app/controllers/users_controller.rb

```diff
@@ -5,8 +5,8 @@ class UsersController < ApplicationController
   before_action :admin_user, only: %i(destroy)
   # GET /users
   def index
-    @pagy, @users = pagy(User.recent, items: Settings.defaults.user.items_per_page, page: params[:page],
-items_param: false)
+    @pagy, @users = pagy(User.all, items: Settings.defaults.user.items_per_page,
```

Comment:
```suggestion
    @pagy, @users = pagy(User.recent, items: Settings.defaults.user.items_per_page,
```
Vẫn dùng sort trong pagy